Author: Andreas Beckmann <anbe@debian.org>
Description: make the initscripts follow Debian standards
Forwarded: not-needed

--- a/init/sysv/nvidia-persistenced.template
+++ b/init/sysv/nvidia-persistenced.template
@@ -34,10 +34,11 @@
 #
 ### BEGIN INIT INFO
 # Provides:         nvidia-persistenced
-# Required-Start:   $ALL
-# Required-Stop:    $ALL
+# Required-Start:   $local_fs
+# Required-Stop:    $local_fs
 # Default-Start:    2 3 4 5
 # Default-Stop:     0 1 6
+# Short-Description: Starts and stops the NVIDIA Persistence Daemon
 # Description:      Starts and stops the NVIDIA Persistence Daemon
 ### END INIT INFO
 
@@ -68,9 +69,9 @@ case "${1}" in
       echo "Stopping NVIDIA Persistence Daemon"
 
       # Stop the daemon - its PID should have been read in
-      [ ! -z "${NVPD_PID}" ] && kill ${NVPD_PID} &> /dev/null
+      [ ! -z "${NVPD_PID}" ] && kill ${NVPD_PID} >/dev/null 2>&1
       ;;
-  restart)
+  restart|force-reload)
       $0 stop
       sleep 2
       $0 start
